scripts/lib/mkcloud-common.sh

```diff
@@ -54,7 +54,8 @@ for port in 22 80 443 5000 7630; do
     done
 done
 
-iptables_unique_rule PREROUTING -t nat -p tcp --dport 6080 \\
+# Forward VNC port
+iptables_unique_rule PREROUTING -t nat -p tcp --dport \$(( $cloud_port_offset + 6080 )) \\
```

Comment:
It could be calculated during script generation as the offset is known at that time but I did it like this for consistency with the rest of the script.